.github/workflows/pipeline.yaml

```
@@ -35,7 +35,7 @@ jobs:
         azcliversion: latest
         inlineScript: |
           az account show
-          echo ${{ secrets.AZURE_DEVOPS_EXT_PAT }} | az devops login --org ${{ secrets.AZURE_PIPELINE_ORG }}
+          export ${{ secrets.AZURE_DEVOPS_EXT_PAT }}
```

does this need to be 
```suggestion
          export AZURE_DEVOPS_EXT_PAT=${{ secrets.AZURE_DEVOPS_EXT_PAT }}
```